src/lib/components/WorkstreamDetail.svelte

```diff
@@ -33,64 +60,148 @@
         >on {dateFormat(workstream.created_at)}</span
       >
     </div>
-    <Card hoverable={false}>
-      <div slot="top">
-        <h3>Applications</h3>
-        <div class="timerate">
-          <div style="text-align: right;">
-            <p class="typo-text-bold rate">
-              {Math.floor(workstream.payment.rate)}
-              {workstream.payment.currency}
-              <span class="typo-regular">/ day</span>
-            </p>
-          </div>
-          {#if workstream.type === 'grant' && workstream.duration}
-            <div>
-              <p class="timeframe">{timeframeFormat(workstream.duration)}</p>
+    {#if workstream.state !== WorkstreamState.ACTIVE}
```

Comment:
not a big deal but I feel like things would be a bit cleaner if you'd do this if condition on === ACTIVE and flip the blocks around. The negative here makes things a bit harder to parse.